frontend/pages/water-stat.js

```diff
@@ -74,10 +76,49 @@ const WaterStat = () => {
             {tool.title}
           </h1>
 
-          <div
-            className="text-gray-800 leading-relaxed"
-            dangerouslySetInnerHTML={{ __html: tool.description }}
-          />
+          <ReactMarkdown
```

Comment:
I think this is getting complex enough that this should be centralised to make the code DRYer